playwright/tests/test.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { test, expect } from '@playwright/test';
+
+test('Open moby-epub3', async ({ page }) => {
+  await page.goto('http://localhost:1234/html/moby-epub3');
```

Comment:
i think we can put `http://localhost:1234` into the Playwright config file somewhere https://playwright.dev/docs/test-configuration